var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -55,9 +56,14 @@ class Adios(AutotoolsPackage):
     variant('zlib', default=True, description='Enable zlib transform support')
     variant('bzip2', default=False, description='Enable bzip2 transform support')
     variant('szip', default=False, description='Enable szip transform support')
-    variant('zfp', default=False, description='Enable ZFP transform support')
+    variant('zfp', default=True, description='Enable ZFP transform support')
+    variant('sz', default=True, description='Enable SZ transform support')
```

Comment:
I can't compile the stable version of sz (listed as v1.4.9-beta in the sz spack package), so am setting the dependency on develop till the sz package is fixed